src/test/TestUtils.h

```diff
@@ -62,7 +62,6 @@ std::shared_ptr<T>
 createTestApplication(VirtualClock& clock, Config const& cfg)
 {
     Config c2(cfg);
-    c2.USE_CONFIG_FOR_GENESIS = true;
```

Comment:
from a different PR (where this change was started), you were going to look into potential failures of some other tests.

> I checked those tests overnight and some of them didn't finished for a long time.
I mean "Mesh nodes vs. network traffic" and "Stress test on 2 nodes 3 accounts 10 random transactions 10tx/sec".
I'll check those tonight without those changes.

Reply:
Both of those tests do not finish on master. I'll open issues for that.